modules/AWS/rds/main.tf

```diff
@@ -25,7 +25,7 @@ module "db" {
   source  = "terraform-aws-modules/rds/aws"
   version = "~> 3.0"
 
-  identifier = var.rds_instance_id
+  identifier = var.rds_instance_identifier
```

Comment:
Just a question, what's the difference between `id` and `identifier`? 
From my view, it doesn't really bring too much value, and even lessened the readability.

Reply:
id is an abstract for identifier, as this is the official term in RDS, I changed this to not get confused with instance_name.